packages/lit-html/src/lit-html.ts

```diff
@@ -262,11 +266,14 @@ export const render = (
   container: HTMLElement | DocumentFragment,
   options?: RenderOptions
 ) => {
-  let part: NodePart = (container as any).$lit$;
+  let part: NodePart = (options?.renderBefore ?? (container as any)).$lit$;
```

Comment:
why does `container` need the cast, but not `options?.renderBefore`?

Reply:
Seems like TS just not noticing that. Fixing.